[DBNBK-207, CRP-1262] add Slack recipients to Scheduled Reports public docs (#27273)
DataDog/documentation: content/en/dashboards/sharing/scheduled_reports.md

    - /dashboards/reporting/
    - /dashboards/scheduled_reports/
further_reading:
    - link: 'https://www.datadoghq.com/blog/dashboard-sharing/'
      tag: 'Blog'
      text: 'Share Datadog dashboards securely with anyone'
    - link: 'https://www.datadoghq.com/blog/template-variable-associated-values/'
      tag: 'Blog'

>       text: 'Use associated template variables to refine your dashboards'

Author: [ℹ️ info] These changes were automatically applied

---

{{< site-region region="gov" >}}

<div class="alert alert-warning">Scheduled Reports are not supported for your selected <a href="/getting_started/site">Datadog site</a> ({{< region-param key="dd_site_name" >}}).</div>
{{< /site-region >}}

## Overview

Scheduled reports enable Datadog users to share dashboards as high-density PDFs on a recurring basis.

{{< img src="dashboards/scheduled_reports/report_pdf.png" alt="Example report PDF attachment" style="width:90%;" >}}

> The report PDF can be sent to Slack or email recipients.

Reviewer: maybe `Slack channels or email addresses` ? since we don't support individual Slack recipients via DM

### 3. Add recipients

#### Email recipients

> Add email recipients to your report by entering their email addresses. The email associated with your Datadog account is automatically added as a recipient. You can remove yourself as a recipient by hovering over your email and clicking the trash icon that appears next to it.

Reviewer 2: ```suggestion
To add email recipients to your report, enter their email addresses. The email associated with your Datadog account is automatically added as a recipient. You can remove yourself as a recipient by hovering over your email and clicking the trash icon that appears next to it.
```

**Note:** Enterprise and Pro accounts can send reports to recipients outside of their organizations.

**{{< img src="dashboards/scheduled_reports/add_email_recipients.png" alt="The configuration modal for editing scheduled report variables." style="width:90%;" >}}**

To see the report before saving the schedule, click **Send Test Email**. You can pause a report schedule at any time.

#### Slack recipients

> Add Slack recipients by selecting the Slack workspace and channel from the available dropdowns. You can send a test message to Slack by adding a channel recipient and click **Send Test Message**.

Reviewer: can you add something about needing to configure the Slack integration? https://docs.datadoghq.com/integrations/slack/?tab=datadogforslack

Author: Added!

> Add Slack recipients by selecting the Slack workspace and channel from the available dropdowns. You can send a test message to Slack by adding a channel recipient and click **Send Test Message**.

Reviewer 2: ```suggestion
To add Slack recipients, select the Slack workspace and channel from the available dropdowns. You can send a test message to Slack by adding a channel recipient and clicking **Send Test Message**.
```

### 3. Add recipients

#### Email recipients

> To add email recipients to your report, by enter their email addresses. The email associated with your Datadog account is automatically added as a recipient. You can remove yourself as a recipient by hovering over your email and clicking the trash icon that appears next to it.

Reviewer 2: ```suggestion
To add email recipients to your report, enter their email addresses. The email associated with your Datadog account is automatically added as a recipient. You can remove yourself as a recipient by hovering over your email and clicking the trash icon that appears next to it.
```

**Note:** Enterprise and Pro accounts can send reports to recipients outside of their organizations.

**{{< img src="dashboards/scheduled_reports/add_email_recipients.png" alt="The configuration modal for editing scheduled report variables." style="width:90%;" >}}**

To see the report before saving the schedule, click **Send Test Email**. You can pause a report schedule at any time.

#### Slack recipients

> To add Slack recipients, select the Slack workspace and channel from the available dropdowns. If you do not see any Slack workspaces available, endure you have the Datadog [Slack Integration](https://docs.datadoghq.com/integrations/slack/?tab=datadogforslack) installed. To send a test message to Slack, add a channel recipient and click **Send Test Message**.

Reviewer 2: ```suggestion
To add Slack recipients, select the Slack workspace and channel from the available dropdowns. If you do not see any Slack workspaces available, ensure you have the Datadog [Slack Integration][8] installed. To send a test message to Slack, add a channel recipient and click **Send Test Message**.
```

Users with the **Org Management** permission can enable or disable the scheduled reports feature for their organization from the **Settings** tab under [Public Sharing][3] in **Organization Settings**.

{{< img src="dashboards/scheduled_reports/report_management_org_preference.png" alt="The Report Management setting in the Settings tab in Public Sharing within Organization Settings in Datadog with the setting Enabled" style="width:90%;" >}}

Additionally, users with the **Org Management** permission can enable or disable Slack recipients for their organization from the **Settings** tab under [Public Sharing][3] in **Organization Settings**.

{{< img src="dashboards/scheduled_reports/report_send_to_slack_org_preference.png" alt="The Send to Slack setting under Report Management setting in the Settings tab in Public Sharing within Organization Settings in Datadog with the setting Enabled" style="width:90%;" >}}

## Unsupported widget types

The following widget types are **not** supported and will be shown as empty in the report:

-   [Iframe][4]
-   [Image][5]
-   [Hostmap][6]

> -   [Run Workflow][7]

Author: [ℹ️ info] These changes were automatically applied

[5]: /dashboards/widgets/image/
[6]: /dashboards/widgets/hostmap/

> [7]: /dashboards/widgets/run_workflow/

Reviewer 2: ```suggestion
[7]: /dashboards/widgets/run_workflow/
[8]: /integrations/slack/?tab=datadogforslack
```
